ci-operator/step-registry/hypershift/aws/create-cluster-byoi/hypershift-aws-create-cluster-byoi-commands.sh

```
@@ -0,0 +1,106 @@
+DEFAULT_BASE_DOMAIN=ci.hypershift.devcluster.openshift.com
+
+HC_REGION=${HYPERSHIFT_AWS_REGION:-$LEASED_RESOURCE}
+
+if [[ $HYPERSHIFT_GUEST_INFRA_OCP_ACCOUNT == "true" ]]; then
+  AWS_GUEST_INFRA_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+  DEFAULT_BASE_DOMAIN=origin-ci-int-aws.dev.rhcloud.com
+fi
+
+DOMAIN=${HYPERSHIFT_BASE_DOMAIN:-$DEFAULT_BASE_DOMAIN}
+RELEASE_IMAGE=${HYPERSHIFT_HC_RELEASE_IMAGE:-$RELEASE_IMAGE_LATEST}
+CLUSTER_NAME="$(cat ${SHARED_DIR}/cluster-name)"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)